core/crypto/rand_darwin.odin

```diff
@@ -1,12 +1,16 @@
 package crypto
 
 import "core:fmt"
-import "core:sys/darwin"
+
+foreign import Security "system:Security.framework"
 
 _rand_bytes :: proc(dst: []byte) {
-	res := darwin.SecRandomCopyBytes(count=len(dst), bytes=raw_data(dst))
-	if res != .Success {
-		msg := darwin.CFStringCopyToOdinString(darwin.SecCopyErrorMessageString(res))
-		panic(fmt.tprintf("crypto/rand_bytes: SecRandomCopyBytes returned non-zero result: %v %s", res, msg))
+	err := SecRandomCopyBytes(count=len(dst), bytes=raw_data(dst))
+	if err != 0 {
+		panic(fmt.tprintf("crypto/rand_bytes: SecRandomCopyBytes returned non-zero result: %v", err))
 	}
 }
+
```

Comment:
There's a little bit of duplication here to prevent importing `CoreFoundation` vendor library in `core:crypto`, which I hope is fine.